docs/devices-api/more-info/webhooks.mdx

```diff
@@ -17,15 +16,72 @@ how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
   page](/devices-api/more-info/our-integrations).
 </Info>
 
-## User data message
+## Provider connected message
+
+Example payload:
+
+```json
+{
+  "meta": {
+    "messageId": "<message-id>",
+    "when": "<date-time-in-utc>",
+    "type": "devices.provider-connected"
+  },
+  "users": [
+    {
+      "userId": "951faef1-5cfd-464a-81f7-31f76edf309e",
+      "provider": "garmin",
+      "connectedProviders": ["fitbit", "garmin", "cronometer"]
+    }
+  ]
+}
+```
 
 The format follows:
 
+<ResponseField name="meta" required>
+  Metadata about the message. The full format is described [here](/home/api-info/webhooks#meta-data).
+
+  <Expandable title="meta type property details">
+
+    <ResponseField name="type" type="DAPIWebhookType" required>
+      The type of the Devices API webhook data message. Example: `devices.provider-connected`.
```

Comment:
I mean for the `Provider connected message`, it will always be `devices.provider-connected`.

Perhaps: "The type of the Devices API webhook data message. This will be set to `devices.provider-connected` for the Provider connected message.